source/usage-examples/transaction.txt

```diff
@@ -0,0 +1,141 @@
+.. _node-usage-convenient-txn:
+
+=====================
+Perform a Transaction
+=====================
+
+The {+driver-short+} provides the Core API and the Convenient
+Transaction API to perform transactions. This usage example provides
+sample code that uses the **Convenient Transaction API**.
+
+.. seealso::
+   
+   To learn about the Convenient Transaction API in more detail, see the
+   :ref:`Convenient Transaction API <node-txn-convenient-api>` section of the
+   Transactions guide.
+
+   To learn about the Core API and view an example, see the
+   :ref:`Core API <node-txn-core-api>` section of the
+   Transactions guide.
```

Comment:
Suggestion:
I think since these link to sections of the same page, it can be easier to treat them together:

"To learn more about the Transactions APIs and see an transaction example that uses the Core API, see the Transactions documentation."

Reply:
not applicable anymore